### PR TITLE
fixed form & add error handler

### PR DESCRIPTION
**Pull request by skymattz**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -223,12 +221,23 @@ <h2>Contact Me</h2>
...
+            // tampilkan tombol kirim, hilangkan tombol loading
+            btnLoading.classList.toggle('d-none');
+            btnKirim.classList.toggle('d-none');
+            // tampilkan alert
+            myAlert.innerHTML = `<strong>Terjadi Kesalahan!</strong> pesan gagal dikirim.`;
+            myAlert.classList.toggle('alert-danger');
+            myAlert.classList.toggle('d-none');
+            // reset formnya
+            form.reset();            
+          });
```
Comment (Dinar10) [inline index.html:240]: Sip

**Review by Dinar10 — COMMENTED** [review]
Mantap

**Review by Dinar10 — APPROVED** [review]

**Comment by skymattz** [comment]
merge euy

**Comment by Dinar10** [comment]
Ok

**Pull request merged**